templates/configmap.yaml

```
@@ -19,3 +20,18 @@ data:
   TAIGA_EMAIL_PORT: {{ default 587 .Values.taiga.emailSmtpPort | quote }}
   TAIGA_EMAIL_USER: {{ required "taiga.emailSmtpUser must be set" .Values.taiga.emailSmtpUser | quote }}
 {{- end }}
+{{- if .Values.taiga.ldap.Enabled }}
```

Same as below, lowercase would be more fitting
```suggestion
{{- if .Values.taiga.ldap.enabled }}
```